features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -198,11 +204,30 @@ data class SessionReplayConfiguration internal constructor(
         }
 
         private fun customMappers(): List<MapperTypeWrapper<*>> {
-            return extensionSupport.getCustomViewMappers()
+            val allItems = extensionSupport.flatMap { it.getCustomViewMappers() }
+            val uniqueMap = mutableMapOf<String, MapperTypeWrapper<*>>()
+
+            allItems.forEach { item ->
+                if (uniqueMap.containsKey(item.type.toString())) {
```

Comment:
How about override `equals`  in `MapperTypeWrapper` implementation by comparing only type, then you don't have to build a map here.